lib/resources/r4/charge_item_modify.yaml

```diff
@@ -0,0 +1,417 @@
+        "valueReference": {
+          "reference": "Location/21304903",
+          "display": "NM"
+        },
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/performing-location"
+      }
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.extension
+
+  - name: description
+    required: 'No'
+    type: Extension
+    description: A description providing additional details of the resource.
+    action: modify
+    example: |
+      {
+        "valueString": "Acetaminophen Level",
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/description"
+      }
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.extension
+
+  - name: custom-attribute
+    required: 'No'
+    type: Extension
+    description: A client defined custom attribute for the charge.
+    action: modify
+    example: |
+      {
+        "extension": [
+          {
+            "id": "2572130093",
+            "valueString": "Special Authority",
+            "url": "custom-attribute-name"
+          },
+          {
+            "valueInteger": 1,
+            "url": "custom-attribute-value"
+          }
+        ],
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/custom-attribute"
+      }
+    note: >
+      <ul>
+        <li>Custom-attribute-name and Custom-attribute-value are both required</li>
+        <li>Custom-attribute-value can be of type integer, string, or decimal</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.extension
+
+  - name: modifier-code
+    required: 'No'
+    type: Extension
+    description: A code providing information about the procedure performed on the patient associated to the resource.
+    action: modify
+    example: |
+      {
+        "extension": [
+          {
+            "valueCodeableConcept": {
+              "coding": [
+                {
+                  "extension": [
+                    {
+                      "valueCoding": {
+                        "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002",
+                        "code": "3692"
+                      },
+                      "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
+                    }
+                  ],
+                  "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-MODIFIER",
+                  "code": "66"
+                }
+              ]
+            },
+            "url": "code"
+          },
+          {
+            "valueUnsignedInt": 1,
+            "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/priority"
+          }
+        ],
+        "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/modifier-code"
+      }
+    note: >
+      <ul>
+        <li>The coding field must contain only one coding from the MODIFIER proprietary Bill Codes Types system.</li>
+      </ul>
```

Comment:
Let's add a note stating that both code and priority sub extensions are required.

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/780/commits/4595890bdc6f918a94d93cd54ca164dab864e8e0#diff-3c565d9c41e2bb3d417efe63a774683e72b07cd38f2ff54a695c695999f29938R321